CRM/Contribute/BAO/ContributionSoft.php

```diff
@@ -618,4 +621,67 @@ protected static function processPCP($pcp, $contribution) {
     }
   }
 
+  /**
+   * Function used to send notification mail to pcp owner.
+   *
+   * @param int $contributionID
+   * @param array $contributionSoft
+   *   Contribution object.
+   *
+   * @throws \API_Exception
+   * @throws \CRM_Core_Exception
+   */
+  public static function pcpNotifyOwner(int $contributionID, array $contributionSoft): void {
```

Comment:
function moved & param switched from contribution to contribution id (I could break this into 2 prs?)